modules/installation-aws-config-yaml.adoc

```diff
@@ -73,6 +73,7 @@ controlPlane: <3> <4>
   name: master
   platform:
     aws:
+      lbType: nlb
```

Comment:
It can either be "NLB" or "Classic". By default if not set it will be "Classic". cc @Miciah